style.css

```diff
@@ -258,10 +262,92 @@ p {
   font-size: 12px;
 }
 
-.recent-works .work-cards .project-one {
-  background-image: url(./icons/Img-Plaholder.png);
-  background-repeat: no-repeat;
-  background-size: cover;
+/* project-modal */
+.desktop-view-btn {
+  display: none;
+}
+
+.mobile-view-btn {
+  display: flex;
+  gap: 12px;
+}
+
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 999;
+  left: 6%;
+  top: 2%;
+  width: 80%;
+  height: 94vh;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  text-align: left;
+  flex-direction: column;
+  padding: 10px;
+  overflow: auto;
+}
+
+.modal .close-modal-btn {
+  align-self: flex-end;
+  border: none;
+  background-color: #fff;
+}
+
+.modal .project-img {
+  align-self: center;
+  display: flex;
+  justify-content: center;
+}
+
+.modal .project-img img {
+  width: 100%;
+  height: 220px;
+}
+
+.project-intro .project-title h2 {
+  font-weight: 600;
+  font-size: 2rem;
+  color: #172b4d;
+}
```

Comment:
Mobile popup:

- [ ] Kindly make sure that when you click on `see project `and a popup appears, the background of the page is blurred and follows the colour displayed as per the Figma design.
- [ ] Kindly make sure you have the correct spacing to the top, right, left and bottom of the popup and all popup items as per the Figma design.
- [ ] Kindly make sure to follow the correct size for buttons and also place them in the centre as per the Figma design.
- [ ] Kindly reduce the spacing between the title of the popup and technologies as per the Figma design.
- [ ] Kindly remove extra technologies as per Figma design.
- [ ] Kindly make sure the popup is displayed in the centre with equal margins. As you can see there is more margin on the right. Fix it.

Your design:
<img width="498" alt="Screenshot 2022-05-11 at 10 35 22 PM" src="https://user-images.githubusercontent.com/70269507/167931604-81e26ea7-fa12-481f-bbec-26cab93d20ed.png">


Figma design:
<img width="170" alt="Screenshot 2022-05-11 at 10 30 51 PM" src="https://user-images.githubusercontent.com/70269507/167931317-def2ae4d-3e5b-423a-9599-d09d25ada001.png">